lib/plugins/aws/package/compile/events/cloudFront/index.js

```diff
@@ -231,9 +248,9 @@ class AwsCompileCloudFrontEvents {
     });
 
     // sort that first is without PathPattern if available
-    behaviors.sort((a, b) => {
+    behaviors = _.sortBy(behaviors, b => {
```

Comment:
Why it was changed to `_.sortBy`? (currently our policy to resort to lodash, only if same operation cannot be achieved natively in clean way)

Reply:
There was a problem with this issue https://github.com/nodejs/node/issues/24294 on node12 because PathPatterns wasnt tested at all and bug wasnt spotted. Ive prepared fixed native sort function.